client/vite.config.js

```diff
@@ -8,9 +8,13 @@ export default defineConfig({
     globals: true,
     setupFiles: './tests/setup.js',
     environment: 'jsdom',
+
+    include: ['src/tests/**/*.{test,spec}.{js,jsx,ts,tsx}'],
+
+    
     coverage: {
       provider: 'istanbul',
-      include: ['src/**/*'],
+      include: ['src/**/*.{test,spec}.{js,jsx}'],
```

Comment:
We should not include the test files in the coverage report, this will give us a false statistic. Kindly keep this line as was before.